clang/test/Driver/mips-features.c

```diff
@@ -214,6 +214,14 @@
 // RUN:     -mnan=legacy -mnan=2008 2>&1 \
 // RUN:   | FileCheck --check-prefix=CHECK-NAN2008 %s
 // CHECK-NAN2008: "-target-feature" "+nan2008"
+// CHECK-NAN2008: "-target-feature" "+abs2008"
```

Comment:
If abs2008 is adjacent, just place it after "+nan2008" so that it's clear there is no other option in between.